docs/guide/custom-ui.md

```diff
@@ -198,11 +198,13 @@ The `AposWidget` component has **nothing to do with a typical site visitor exper
 Before you override an editor modal, consider [adding a custom schema field type](/guide/custom-schema-field-types.md) instead.
 :::
 
-## Adding custom context menu
+## Adding custom context menu items
 
-Starting from Apostrophe v3.18.0 we can add custom context menus (edit mode) from within any module, targeting any Vue component that implements `AposModal`. The menu registration should happen in the initialization phase.
+We can add custom context menu items (edit mode) from within any module, targeting any Vue component that implements `AposModal`. The menu registration should happen in the initialization phase.
```

Comment:
I think we need to define what the context menu is first, in words (although the image is of course very helpful).

I suggest:

"Apostrophe offers a context menu that can be used to carry out certain operations on a document, such as 'preview', 'duplicate', and so on."

Reply:
In the screenshot, "My Menu" needs to change to "My Menu Item".